### ISSUE TITLE
Add a property for PackageBase class related to wrappers

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1477]
In GitLab by @dbeltrankyl on Dec 2, 2024, 16:41

I had to fix one issue for the dev release and found some checks that should be changed.

Related to https://earth.bsc.es/gitlab/es/autosubmit/-/merge_requests/525/diffs?commit_id=866119c6ba070ab9b006fa82a45b320c4cca9e19

The idea is to add the @property, package.contains_a_wrapper to substitute some band-fixes comparations that rely on the `package.name` being set. 

How to reproduce the error:

Try to add a `package.name` to the `PackageSimple.__init__` class and run the testing suite

**Comment by dbeltrankyl** [comment]
@kinow @VindeeR I think this is the "self.name" stuff you were commenting upon in the meeting, isn't it?

**Comment by VindeeR** [comment]
@dbeltrankyl yes that was the variable I was talking about earlier, at some point it starts to give me some errors and don't allow the workflow finish
